dla_cnn/data_model/DesiMock.py

```diff
@@ -0,0 +1,88 @@
+from astropy.io import fits
+import numpy as np
+
+
+class DesiMock:
+    """
+    a class to load all spectrum from a mock DESI data v9 fits file, each file contains 1186 spectra.
+    :attribute wavelength array-like, the wavelength of all spectrum (all spectrum share same wavelength array)
+    :attribute data, dict, using each spectra's id as its key and a dict of all data we need of this spectra as its value
+      its format like spectra_id: {'FLUX':flux,'ERROR':error,'z_qso':z_qso, 'RA':ra, 'DEC':dec, 'DLAS':dlas information} ,
+      dla information is a tuple, and its format is like (dla_id, z_qso, NHI)
+    :attribute split_point_br, the length of the flux_b,the split point of  b channel data and r channel data
+    :attribute split_point_rz, the length of the flux_b and flux_r, the split point of  r channel data and z channel data
+    :attribute data_size, the point number of all data points of wavelength and flux
+    """
+
+    def _init_(self, wavelength = None, data = {}, split_point_br = None, split_point_rz = None, data_size = None):
+        self.wavelength = wavelength
+        self.data = data
+        self.split_point_br = split_point_br
+        self.split_point_rz = split_point_rz
+        self.data_size = data_size
+
+    def read_fits_file(self, spec_path, truth_path, zbest_path):
+        """
+        read Desi Mock spectrum from a fits file, load all spectrum as a DesiMock object
+        :param spec_path: spectrum file path

```

Comment:
Indicate that each of these paths is a `str`

Reply:
Have done.